src/scancode/cli.py

```diff
@@ -1176,10 +1176,11 @@ def run_scanners(
 
     progress_manager = None
     if not quiet:
+        resources = list(((r.location, r.path) for r in codebase.walk() if r.is_file))
         echo_func(f'Scan files for: {scan_names} with {processes} process(es)...')
         item_show_func = partial(path_progress_message, verbose=verbose)
         progress_manager = partial(progressmanager,
-            item_show_func=item_show_func,
+            item_show_func=item_show_func, length=len(resources)*2,
```

Comment:
Why the times 2?

Reply:
Because https://github.com/aboutcode-org/scancode-toolkit/issues/3344 Otherwise the progressbar go to 170/170 and then continue to 340/170. This *2 can be removed when we find when we find where is the count done twice. I have a suspicion, but it needs more testing.